func/ip.sh

```diff
@@ -62,13 +64,13 @@ update_ip_helo_value() {
 
     #Create or update ip:helo pair in mailhelo.conf file
     if [ ! -z "$helo" ]; then
-        if [ $(cat /etc/${MAIL_SYSTEM}/mailhelo.conf | grep "$ip") ]; then
-            sed -i "/^$ip:/c $ip:$helo" /etc/${MAIL_SYSTEM}/mailhelo.conf
+        if [ $(cat /etc/${MAIL_SYSTEM}/mailhelo.conf | grep "$natip") ]; then
+            sed -i "/^$natip:/c $natip:$helo" /etc/${MAIL_SYSTEM}/mailhelo.conf
         else
-            echo $ip:$helo >> /etc/${MAIL_SYSTEM}/mailhelo.conf
+            echo $natip:$helo >> /etc/${MAIL_SYSTEM}/mailhelo.conf
         fi
     else
-        sed -i "/^$ip:/d" /etc/${MAIL_SYSTEM}/mailhelo.conf
+        sed -i "/^$natip:/d" /etc/${MAIL_SYSTEM}/mailhelo.conf
     fi
 }
```

Comment:
Don't we want the real ip here?

Reply:
We want the public ip to be visible or not? Local  ip doesn't make any sense

Reply:
https://github.com/hestiacp/hestiacp/blob/47cf4a076776c225d686b08c7aab94234d76c535/bin/v-change-sys-hostname#L65-L69

If we run v-change-sys-hostname dev.hestiacp.com the public ip pass trough will be 1.2.3.4 

https://github.com/hestiacp/hestiacp/blob/4a8b2294d407c52e47030dc7234f3c609596770b/func/ip.sh#L48-L55

Currently this "$ip" value creates an new config file for "1.2.3.4" 

"ip" pass trough in the system becomes both $ip and $natip  and get overwritten for the local ip 192.168.1.10 for example

For Help we still need to announce the public ip so far I can understand if we use 192.168.1.10 the DNS records will not match the sending domain

Reply:
> We want the public ip to be visible or not? Local  ip doesn't make any sense
> 
> https://serverfault.com/questions/305925/what-exactly-should-helo-say

Yes, we want the public IP.

Reply:
I think I'm not following your changes because it says "In case the IP is an NAT use the real ip address". Okay.

So that check does, if the real ip file doesn't exist, get the real ip and put it in $ip variable. This I follow.

But then later you put the $natip variable into the mailhelo.conf file. Which might not be the public ip.

Reply:
We should save the HELO "value" for /usr/local/hestia/data/ips/local.ip other wise users are allowed to select the public ip and causing issues 

For mailhelo.conf we have to use the public ip. And is set with the current changes. 
When v-change-sys-ip-helo is called for any reason the value set will update in mailhelo.conf the public ip and for Hestia the local ip... 

I think we are safe now

Reply:
Yes, I just miss understood the roles of the variables.